modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/DropdownActionsTag.java

```diff
@@ -25,6 +25,7 @@
 	public int doStartTag() throws JspException {
 		setAttributeNamespace(_ATTRIBUTE_NAMESPACE);
 
+		setCssClass("component-action");
```

Comment:
Doing this we're overwriting any custom `cssClass` passed to the class. For example:

```
<clay:dropdown-actions
   cssClass="my-awesome-class"
/>
```

In this example `my-awesome-class` will be ignored and replaced with `component-action`

Reply:
Ok, I'll change it 👍 Thanks!